### ISSUE TITLE
Basic POI info pane (place page) takes too much space

### ISSUE THREAD

**Issue by pastk** [organicmaps/organicmaps#7332]
![Screenshot_20240208-130713.jpg](https://github.com/organicmaps/organicmaps/assets/18434508/7c390a4b-7b78-4007-92b1-367537cdd18b)

Here it takes two lines.
It used to be just one line before the addition of the share and close buttons.

E.g. in this case I select the POI just to know the approx distance towards it. Its important to see as much map as possible still.

Any ideas how to reclaim this "wasted" space?

**Comment by pastk** [comment]
FYI @Jean-BaptisteC

**Comment by Jean-BaptisteC** [comment]
Caused by #6776 and improved by #7145

**Comment by Misalf-git** [comment]
![place-page_very_interesting.png](https://github.com/organicmaps/organicmaps/assets/72605561/38ff6f42-c5c5-4b1a-a9e1-dd8a3a3edb15)

**Comment by pastk** [comment]
@Jean-BaptisteC almost any nameless POI will do, e.g. 41.860873, 2.664371

I like the proposition of @Misalf-git

**Comment by Jean-BaptisteC** [comment]
Move the blue arrow is not a good idea, some users touch the middle of place page to close it. I'm afraid user touch arrow when he want close place page.
But I will see to reduce again margin of buttons.

**Comment by pastk** [comment]
> Move the blue arrow is not a good idea, some users touch the middle of place page to close it. I'm afraid user touch arrow when he want close place page. 

Good point! But the "share" and "close" buttons should create a similar mis-tap issue? But looks like they don't.
I think users will quickly recognise that tapping on the blue arrow does something else and they'll re-learn to tap on the POI title instead.

Also currently tapping on the title/middle of the place page first expands it, then the second tap closes it.. I don't know if this behavior is as intended, but I find it a bit confusing...

**Comment by Misalf-git** [comment]
Indeed, I only had screen estate in mind.

After playing with the place page to see where I might imaginarily mis-tap, I don't really see a problem, though. Expanding/Closing the pp is possible by swiping up/down (incl. over buttons) and by tapping the (possibly partially unused) title area.

In theory, I agree that "more map" is better than "large button size" if the latter claims space that's otherwise usable for infirmation.

**Comment by biodranik** [comment]
1. Moving the same controls dynamically depending on a title length is a bad idea for muscle memory.
2. Share and close buttons are now also used by Google, not only by Apple. So it's a familiar pattern for many, and it doesn't make sense to change it.

Some other ideas are needed...

**Comment by pastk** [comment]
> 1. Moving the same controls dynamically depending on a title length is a bad idea for muscle memory.

Well e.g. we change positions of bottom bar buttons depending on their number... Doesn't seem to be an issue there, although these buttons are used frequently and their position is fixed vertically.
Whereas the top line of the PP (which is sensitive to closing/expanding taps) could appear at different vertical positions (depending on PP contents) and hence a user will have to look at the screen anyway to make a tap.

**Comment by Jean-BaptisteC** [comment]
Your case is not a main case. In your case you have just the type of POI not a name.
In the most of case, we have his name, his type, his address -> In this case by default we have 3 lines used to show information
